lib/core.ps1

```diff
@@ -571,6 +571,9 @@ function Invoke-ExternalCommand {
         [Parameter(ParameterSetName = "UseShellExecute")]
         [Switch]
         $RunAs,
+        [Parameter(ParameterSetName = "UseShellExecute")]
```

Comment:
This is for `UseShellExecute`, and what's normal case?

Reply:
I mean where we use `CreateNoWindow` before.

Reply:
We don't need `CreateNoWindow` any more since it does not affect a GUI app. As for a console app, no **new console window** is actually created, regardless of the `CreateNoWindow` value.

Reply:
The only limitation is that we cannot use `-Silent` together with `-LogPath` since the latter requires `UseShellExecute = $false`.

Reply:
<img width="440" alt="CleanShot 2023-01-20 at 22 50 08@2x" src="https://user-images.githubusercontent.com/5832170/213727636-fba4a5f9-b89c-415b-b6da-0853bc002f89.png">

It is using `/lwe` for `msiexec` here.

Reply:
@niheaven I know it but this cannot go with the new `-Silent` switch. Any current app manifests will not be broken even if this PR is merged. Any new app manifests can just use a `/l*` flag (e.g., `/lwe`) in `msiexec`'s arguments if `-Silent` is also needed.

Reply:
Shall we move on?

Reply:
One or two days after I could review it?

Reply:
🤣 Will review now.